src/System.Collections/src/System.Collections.builds

```diff
@@ -4,21 +4,8 @@
   <ItemGroup>
     <Project Include="System.Collections.csproj" />
     <Project Include="System.Collections.csproj">
-      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
I'm still not sure if we want to remove this, as I've said earlier in this PR this assembly does have some implementation with it, so by removing the x-compilation for netstandard1.3 we are basically removing (or making it really hard ) the capability of bug fixing for netstandard1.3. @ericstj 's advice was to only remove these whenever dealing with full facades.